common/version

```diff
@@ -0,0 +1 @@
+local1560363052
```

Comment:
Did you mean to add this file? I think this is just created as part of running migrations locally...

Reply:
It was created when I ran the migrations locally. I thought it was required, will remove it.